ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.10.yaml

```diff
@@ -53,7 +53,8 @@ tests:
     cluster_profile: azure-qe
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
-    workflow: cucushift-installer-rehearse-azure-upi-disconnected
+      SIZE_VARIANT: vmgenv1
```

Comment:
We always need to create a dedicated sub folder in `cucushift/installer/rehearse` to save install workflow, to help others get know which install workflow is ready to run integration. Pls create dedicated install workflow, provision/deprovision chain files there.